test/functional/misc/vector_api_cli_test.go

```diff
@@ -25,7 +25,7 @@ var _ = Describe("[Functional][Misc][API_CLI] Functional test", func() {
 	Context("invoking vector CLI commands that talk to the vector API", func() {
 		It("should work", func() {
 			Expect(framework.Deploy()).To(BeNil())
-			out, _ := framework.RunCommand(constants.CollectorName, `curl`, `-sv`, `-m`, `5`, `--connect-timeout`, `3`, `http://127.0.0.1:8686/health`)
+			out, _ := framework.RunCommand(constants.CollectorName, `curl`, `-sv`, `-m`, `5`, `--connect-timeout`, `3`, `http://127.0.0.1:24686/health`)
```

Comment:
Why did we change the port number?

Reply:
Port already in use here: 
https://github.com/openshift/cluster-logging-operator/blob/d8082aa76fb591fd5e439cc2441172b4a1ed7fef/cmd/main.go#L85